schemas/bursar-export/legacy/bursarLegacyExportTypeSpecificParameters.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Did you mean to have these in this PR?

Also, please make sure your file/type names are consistent, if possible (`bursarExportLegacy`, `bursarLegacyExport`, `legacyBursarFeeFines`)

Reply:
Oh, that's weird, I thought I had those merged last week. And yeah, I'll make sure the names are consistent. I just didn't want to have redundancy like `bursarLegacyBursarFeeFinesTypeMapping` I'll just prepend everything with `legacy`, should be fine since they are all under the bursar-export folder anyway

Reply:
I thought so too! Git can be weird sometimes